src/GlobalErrorBoundary.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as branding from './branding'
+
+class ErrorMessages {
+  static TITLE = 'We are currently having some issues with VM portal.'
+  static DESCRIPTION = 'Please refresh page or log out and log back in. If the issue persists, please report a bug on GitHub: '
```

Comment:
These messages can go in **src/intl/messages.js**.

Then you can access it like:
````js
import { msg } from './intl'

const title = msg.globalErrorBoundaryTitle()
const desc = msg.globalErrorBoundaryDescription()
````